### PR TITLE
complete the task

### PR DESCRIPTION
**Pull request by barsuchellos**
[DEMO LINK](https://barsuchellos.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +16,114 @@ export const goods = [
...
+              { this.state.selectedGood }
+              {' '}
+              is selected
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => removeGood()}
+              />
```
Comment (jojnajbude) [inline src/App.tsx:57]: Button should has separate condition for visibility.

**Review by jojnajbude — APPROVED** [review]
That's why your tests don't work.

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +16,114 @@ export const goods = [
...
+            <h1 className="title is-flex is-align-items-center">
+              { this.state.selectedGood }
+              {' '}
+              is selected
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => removeGood()}
```
Comment (Denchepornyuk) [inline src/App.tsx:56]:
```suggestion
                onClick={removeGood}
```

File: src/App.tsx (hunk 1 above)
Comment (Denchepornyuk) [reply src/App.tsx:57]: agree, it's better to move button in a separate component

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**